### PR TITLE
Click button helpers [SCI-4592]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-4592](https://biosistemika.atlassian.net/browse/SCI-4592)

Note:
I left old step definitions commented in the code. Will be removed later when we clean all `.feature` files.

### CODE REVIEW

File: features/result.feature [hunk 1]
```diff
@@ -14,473 +14,480 @@ Background:
...
+ And I click button with id "dropdownMenu1"
  And I click "Delete" link within dropdown menu
  And I click to OK on confirm dialog
  Then I should see "Sucessfully removed result MED from task Experiment design." flash message
+
+ # We need to optimize some tests... When you deleting archived result, we always adding result. This should be initial
+ # state there, already
+
+
+ #### JUST TESTING, will be removed
```
Comment (aignatov-bio) [inline features/result.feature:477]: When it will be removed?

File: features/step_definitions/shared_steps.rb [hunk 2]
```diff
@@ -11,23 +11,44 @@
...
+
+# Given('I click on {string} class button') do |button6|
+#   find('.btn', class: button6, match: :first).click
+# end
+
+Then('I click element with css {string}') do |selector|
+  find(selector).click
 end
 
-Given('I click on {string} id button') do |button1|
```
Comment (aignatov-bio) [inline features/step_definitions/shared_steps.rb:30]: Where we need it?

Comment (urbanrotnik) [reply features/step_definitions/shared_steps.rb:30]: What you mean exectly? `Given('I click on {string} id button') do |button1|` was removed?

File: features/result.feature (hunk 1 above)
Comment (urbanrotnik) [reply features/result.feature:477]: Samo will remove it. It's there for demonstration.

Comment (aignatov-bio) [reply features/result.feature:477]: OK

**Pull request merged**